libpromises/process_lib.h

```diff
@@ -27,7 +27,12 @@
 
 #include <platform.h>
 
-#define PROCESS_START_TIME_UNKNOWN ((time_t)0)
+
+/* TODO move to libutils, once windows implementation is merged. */
+
```

Comment:
@kacfengine this is an example of what I was describing. One function implemented in libcfenterprise (`GracefulTerminate()`) keeps all these files (`process_*.[ch]`) in libpromises, instead of libutils where they belong.